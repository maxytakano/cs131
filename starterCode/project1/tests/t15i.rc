structdef MS {
  float a;
};

function : void main()
{
  float* b;
  MS* d;
  (*d).a = *b;
  d->b = *b;
}
