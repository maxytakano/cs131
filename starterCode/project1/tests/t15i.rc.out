Error, "/tmp/cs131_testrunner_QHTaux/file.rc", line 10: 
  Referenced field b not found in type MS.
Compile: failure.
